src/redux/reducers/getArticlesReducer.js

```diff
@@ -1,7 +1,7 @@
 import { GET_ARTICLES } from '../actions/actionTypes';
 
 export const initialState = {
-  articles: [],
+  articles: {},
```

Comment:
We are using one Initial state but  I can see in this file you are not importing it why?

Reply:
this one is used for articles not for comments